src/StepperFirmata2.cpp

```diff
@@ -87,7 +87,7 @@ boolean StepperFirmata2::handleSysex(byte command, byte argc, byte *argv)
   if (command == ACCELSTEPPER_DATA) {
 
     byte stepCommand, deviceNum, interface, wireCount, stepType;
-    byte stepOrMotorPin1, directionOrMotorPin2, motorPin3, motorPin4, enablePin, invertPins = 0;
+    byte stepOrMotorPin1, directionOrMotorPin2, ms1OrMotorPin3, ms2OrMotorPin4, enablePin, invertPins = 0;
```

Comment:
All the driver style boards I can find are derivatives of the easyDriver, but I did some digging and found that there are even versions of the easyDriver (v3 and earlier) that do not have the MS pins. Also, the config section became one of those things that would be infuriatingly obfuscated for future maintainers.

It was neat to see it working, but I think I'm on board with moving the functionality out of here and into either firmata.js or johnny-five. Since J5 is purpose built to normalize controller and device interfaces I think that's a good place for it and we wouldn't need to add any whacky logic to firmata.js.